Makefile

```diff
@@ -83,6 +83,11 @@ manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and Cust
 	$(CONTROLLER_GEN) \
 		rbac:roleName=manager-role output:rbac:artifacts:config=config/components/rbac\
 		paths="./api/..."
+
+	$(CONTROLLER_GEN) \
```

Comment:
I noticed that I wasn’t generating crds anymore. Added this allows me to generate the CRD.

not sure when this broke.